microsoft-edge/webdriver-chromium.md

```diff
@@ -183,6 +214,19 @@ service_args = ['--verbose']
 driver = Edge(service_args = service_args)
 ```  
 
+#### [JavaScript](#tab/javascript/)  
+
+<a id="customizing-microsoft-edge-driver-services-code" />  
+
+When using JavaScript, the `edge` object has a `ServiceBuilder` method that returns a `service` you may optionally pass to the `edge.driver` object, which creates and starts the service for you. 
+
+To configure the `EdgeService`, run additional methods within `edge.ServiceBuilder()` before calling the `build()` method and then pass the service as a parameter in the `edge.Driver.createSession()` method.
```

Comment:
`EdgeService` isn't a thing. This should just be `Service`. Also please remove the `edge.` prefixes here too.